examples/.isort.cfg

```diff
@@ -1,12 +0,0 @@
-[settings]
-skip_glob = **/__init__.py
-default_section = THIRDPARTY
-ensure_newline_before_comments = true
-skip = tutorials/core_api_pytorch_mnist/model_def_distributed.py
```

Comment:
wait, we just added this... it looks like it's being removed with this change?

Reply:
or... that's probably what the `profile=black` bit does I suppose

Reply:
Yes, the `profile = black` includes setting `ensure_newline_before_comments`, and the `skip` doesn't appear to be necessary as long as one of those is present.